### DISCUSSION TITLE
How do I change the colors of the tabs and Form.item's??

### DISCUSSION THREAD

**Discussion by Vamonosraza** [ant-design/ant-design#42360] [Q&A] [answered]
I have a dark background and would like to change the font color, how do I do this? May you give me directions, thanks!
`const items: TabsProps['items'] = [
  {
    key: '1',
    label: `Tab 1`,
    children: `Content of Tab Pane 1`,
  },
  {
    key: '2',
    label: `Tab 2`,
    children: `Content of Tab Pane 2`,
  },
  {
    key: '3',
    label: `Tab 3`,
    children: `Content of Tab Pane 3`,
  },
];`

**Comment by BoyYangzai** [comment, marked as answer]
@Vamonosraza Refer to https://codesandbox.io/s/nostalgic-morse-sdu4f0?file=/index.js
1. you can use `ConfigProvider` and use Design Token
2. Modify the css as you would a traditional `ClassName` or `Style`

**Reply by Vamonosraza** [reply]
Thanks! It did work. However, the text is now unaligned from the placeholders. It is due to the ConfigProvider. I went over it through Inspect and I have to check off 'text-align: end'. How can I fix this? Thanks a lot!

<img width="1344" alt="Screenshot 2023-05-14 at 11 48 07 PM" src="https://github.com/ant-design/ant-design/assets/89946548/bf086dab-ff74-4562-8ffe-f799ea6ebd31">

**Reply by Vamonosraza** [reply]
Nevermind, I got it! Thanks!

**Reply by Vamonosraza** [reply]
I just noticed that the text in the Input Box is also White now. How can I Keep the outside color white but the inside black?

**Reply by BoyYangzai** [reply]
@Vamonosraza 
Hello ,the problems above don't seem to be caused by the `ConfigProvider`
The `ConfigProvider` essence is that `useContext` will not change your layout

**Reply by BoyYangzai** [reply]
@Vamonosraza 
https://ant.design/docs/react/customize-theme#local-theme
cc
use `ConfigProvider` locally , like `useContext`
